qiskit_nature/operators/second_quantization/fermionic_op.py

```diff
@@ -358,6 +371,36 @@ def _to_sparse_label(label):
             filter(None, (label_transformation[c].format(i=i) for i, c in enumerate(label)))
         )
 
+    def to_normal_order(self) -> "FermionicOp":
+        """Convert to the normal order. The returned operator is a sparse label mode."""
```

Comment:
This does not convert self right? it returns a new sparse label FermionicOp that is converted to normal mode.

Why does it change self though - it sets `self.sparse_label = False` which to me looks wrong if its building out a complete new one based on the current, it should not affect/change the current one i.e. self though right?

Reply:
Thanks. This is the bug.This method should not change the `self.sparse_label`. This is only for the conversion.